src/CMakeLists.txt

```diff
@@ -129,9 +130,10 @@ set_target_properties( geosx PROPERTIES INSTALL_RPATH_USE_LINK_PATH TRUE )
 
 install(TARGETS geosx RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
 
+set(SCHEMA_DIR ${CMAKE_SOURCE_DIR}/coreComponents/schema)
```

Comment:
Why do you extract `SCHEMA_DIR` from the next `if` scope?
It looks to me that it's not used outside of this scope?

Reply:
you are true, previous attempt I didn't rollback